src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { Todo } from '../../types/Todo';
+import { FilterStatus } from '../../types/FilterStatus';
+import cn from 'classnames';
+
+interface Props {
+  todos: Todo[];
+  selectedFilter: FilterStatus;
+  setSelectedFilter: (selectedFilter: FilterStatus) => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  selectedFilter,
+  setSelectedFilter,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed).length;
+  const completedTodos = todos.some(todo => todo.completed);
```

Comment:
It's better to name this constant something like `hasCompletedTodos`, so that the name describes this constant the most precisely.